### ISSUE TITLE
For child topic inside docs, should be given tab for better accessibility

### ISSUE THREAD

**Issue by HendryTju** [codescapes-io/codescapes-io.github.io#29]
1. Go to Docs
2. Click on Topics
3. Website will shows child topic(s)

![Current](https://user-images.githubusercontent.com/39986552/157357625-e911b010-f3b6-4d41-bc01-ead8f34ef753.png)

As you can see, the child topic(s) is on same alignment with topic. Should give it more tabs.
